desci-models/src/ResearchObject.ts

```diff
@@ -221,7 +218,7 @@ export interface DataBucketComponentPayload {
 
 export interface PdfComponent extends ResearchObjectV1Component {
   type: ResearchObjectComponentType.PDF;
-  subtype?: ResearchObjectComponentDocumentSubtype | ResearchObjectComponentDocumentSubtypeV2;
```

Comment:
won't this break backwards compatibility if 'other' was selected? @Electromorphous